sirepo/feature_config.py

```diff
@@ -110,6 +110,26 @@ def proprietary_sim_types():
     )
 
 
+def _is_fedora_36():
+    from pykern import pkio
+    import subprocess
+
+    p = pkio.py_path("/etc/os-release")
+    if not p.check():
+        return False
+    for f, v in ("${ID,,}", "fedora"), ("$VERSION_ID", "36"):
```

Comment:
I don't think this is a good approach. Rather, make some hard assumptions and use a simple regex. /etc/os-release is in a particular format on fedora 36, e.g.

```py
"fedora:36" in p.read()
```

I think this is robust enough for the particular problem here.

Reply:
c6a354ba0